File: go-client-sdk/pkg/models/shared/contactupdate.go
package shared

// ContactUpdate
// Update a Contact.
type ContactUpdate struct {
	Additional  map[string]interface{} `json:"additional,omitempty"`
	Email       *string                `json:"email,omitempty"`
	FirstName   *string                `json:"firstName,omitempty"`
	JobTitle    *string                `json:"jobTitle,omitempty"`
	LastName    *string                `json:"lastName,omitempty"`
	MobilePhone *string                `json:"mobilePhone,omitempty"`
	Phone       *string                `json:"phone,omitempty"`
}

File: typescript-client-sdk/src/sdk/models/shared/contactupdate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// ContactUpdate
/** 
 * Update a Contact.
**/
export class ContactUpdate extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=additional" })
  additional?: Record<string, any>;

  @SpeakeasyMetadata({ data: "json, name=email" })
  email?: string;

  @SpeakeasyMetadata({ data: "json, name=firstName" })
  firstName?: string;

  @SpeakeasyMetadata({ data: "json, name=jobTitle" })
  jobTitle?: string;

  @SpeakeasyMetadata({ data: "json, name=lastName" })
  lastName?: string;

  @SpeakeasyMetadata({ data: "json, name=mobilePhone" })
  mobilePhone?: string;

  @SpeakeasyMetadata({ data: "json, name=phone" })
  phone?: string;
}
